com.microsoft.mrtk.uxcomponents/Buttons/Canvas/Action Button.prefab

```diff
@@ -875,6 +875,10 @@ PrefabInstance:
       propertyPath: m_LocalEulerAnglesHint.z
       value: 0
       objectReference: {fileID: 0}
+    - target: {fileID: 3705378105823492766, guid: b85e005d231192249b7077b40a4d4e45, type: 3}
```

Comment:
What are these changes? We should avoid extraneous overrides and changes as this is one of the most critical prefabs to a lot of our downstream customers.

Reply:
Looks like it was not fully reverted somehow when removing SISI label. Completely reverted now and no changes to Action Button.prefab in this PR.